manifests/0000_30_cluster-api_13_vap.yaml

```diff
@@ -0,0 +1,121 @@
+apiVersion: admissionregistration.k8s.io/v1
```

Comment:
As we talked about on that Slack thread, we can't deploy these like so, we'll need to put them in a transport configmap for conditional deploy based on the MAPI Migration feature gate.